### ISSUE TITLE
Likers: To `tabindex="-1"` or to not?

### ISSUE THREAD

**Issue by LeaVerou** [LeaVerou/bluesky-likes#11]
Currently, all profiles are regular links, which means that in the worst case keyboard users have to tab through 101 (!) links to escape this component. This is _terrible_!

If we add `tabindex="-1"` this takes them out of the flow but then they are not accessible. However, if the final link remains accessible, perhaps that’s okay, since it contains a list of all of them? And their avatars still are regular images with regular titles. It's not like the links are essential content, in Selma's original widget there were no links at all.

But actually, thinking about this even more, what are the use cases for using this widget entirely on its own, with no context or other links? Typically you'd use it alongside a `<bluesky-likes>` or other similar UI. So perhaps all of these links can have a `tabindex="-1"`. And for the odd case where you are using it on its own and truly need some kind of fallback content, perhaps we could have a visually hidden link in the default slot 💡
